ot/utils.py

```diff
@@ -492,6 +492,79 @@ def get_coordinate_circle(x):
     return x_t
 
 
+def reduce_lazytensor(a, fun, axis=None, nx=None, batch_size=None):
+    """ Reduce a LazyTensor along an axis with function fun using batches.
```

Comment:
the code it's just for 2d arrays/tensors. Do you confirm?

Reply:
updated the doc, t should work for any shape but reuction is imllemented only for global and axis 1 and 2